index.js

```diff
@@ -428,6 +428,7 @@ class NYC {
       reports.create(_reporter, {
         skipEmpty: this.config.skipEmpty,
         skipFull: this.config.skipFull,
+        relative: this.config.relative === 'true',
```

Comment:
```suggestion
        cwd: this.cwd,
```

Reply:
Hi @coreyfarrell 
Shouldn't I use this.config.cwd to access the cwd option ?

Reply:
No, please use `this.cwd`.  In some edge cases it's possible that `this.config.cwd` might be unset but this.cwd has the default `process.cwd()`.

https://github.com/istanbuljs/nyc/blob/master/index.js#L60

Reply:
Perfect I didn't see it was already set. I do it tonight
Thank you for your anwser

Reply:
I've altered the suggested change per my latest comment on the istanbuljs PR.